hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/ObjectDataInputStreamFinalMethodsTest.java

```diff
@@ -132,8 +132,8 @@ public void testReadUnsignedByte() throws Exception {
 
     @Test
     public void testReadUnsignedShort() throws Exception {
-        inMockedDis.readUnsignedShort();
-        verify(mockedDis).readUnsignedShort();
+        inMockedDis.readShort();
```

Comment:
As the test method name suggest, we should probably leave `readUnsignedShort` call here.

Reply:
Thanks, fixed